### ISSUE TITLE
Add information about docker images hosted on github

### ISSUE THREAD

**Issue by shadowoff09** [badges/shields#8950]

**Comment by calebcartwright** [comment]
It's not exactly clear what you're requesting, but curious whether you've reviewed our docker docs at https://github.com/badges/shields/blob/master/doc/self-hosting.md#docker

**Comment by shadowoff09** [comment]
im saying put the info in this page (https://github.com/shadowoff09/cinema-quotes/pkgs/container/cinema-quotes) in badges, number of downloads, last update and so on

**Comment by calebcartwright** [comment]
Sorry but it's still unclear as to what you are asking. If this is a feature request to provide badges for artifacts hosted in [GitHub's Package Registry](https://github.com/features/packages), then we can close this issue because it's a duplicate of several existing ones (e.g. #4183, #4071, #3608, #5594, etc. all still blocked on #4169).

If you're asking for something different then it'd be great if you could elaborate more thoroughly, preferably providing the data points we request as part of our [issue template for new badge requests](https://github.com/badges/shields/issues/new?assignees=&labels=service-badge&template=3_Badge_request.yml).

**Comment by calebcartwright** [comment]
Closing, will reconsider reopening if new information/clarity is provided
